postprocess/update_images.jsonnet

```diff
@@ -0,0 +1,37 @@
+/**
+ * Adjust generated yamls by helm template
+ */
+local com = import 'lib/commodore.libjsonnet';
+local inv = com.inventory();
+local params = inv.parameters.stackgres_operator;
+
+local file_extension = '.yaml';
+local file_names = [
+  'create-certificate-job',
+  'set-crd-version-job',
+  'wait-job',
+  'wait-webhooks-job',
+];
+
+local image = '%s/%s:%s' % [ params.images.kubectl.repository, params.images.kubectl.image, params.images.kubectl.tag ];
+
+{
+  [file]: com.yaml_load(std.extVar('output_path') + '/' + file + file_extension) {
```

Comment:
what is the reason we are changing the image version from the default version of the chart?

Reply:
so, we had an issue downloading the image (pull rate limit) and decided to make a post processing to point it to our cache.